ocaml-variants.opam

```diff
@@ -11,7 +11,7 @@ depends: [
 conflict-class: "ocaml-core-compiler"
 flags: compiler
 build: [
-  ["./configure" "--prefix=%{prefix}%"]
+  ["./configure" "--prefix=%{prefix}%" "--docdir=%{prefix}%/doc/ocaml"]
```

Comment:
This should be `--docdir=%{ocaml:doc}%`(or `--docdir=%{_:doc}%` but I can't find it in the documentation, I'm no longer sure if that's the right syntax to not mention the package name in the `opam` file).

Reply:
What's the actual path which is wanted? `%{_:doc}%` should install it to `doc/ocaml-variants` in the switch (so it would be `doc/ocaml-base-compiler` in a released compiler). Does this difference matter? I'm not sure if `%{ocaml:doc}%` works at this stage (it shouldn't, because the `ocaml` package is installed _after_ `ocaml-variants`, but it might because `post` dependencies are weird).

Reply:
Oof, I'm so ignorant about `opam` :(

I *think* that what we want is to install it in `${prefix}/doc/ocaml`. This last change was an attempt at producing this exact path without hardcoding it. @dbuenzli do you agree?

Reply:
> What's the actual path which is wanted?

You really want them in `opam var ocaml:doc`, I didn't know about these packaging convolutions. So maybe what @nojb had written in the first place is best. (Sorry for the noise).

Reply:
> So maybe what @nojb had written in the first place is best. (Sorry for the noise).

Actually rather than what he wrote originally `%{doc}%/ocaml/` would be better.

Reply:
Sounds good, amended.

Reply:
Not ignorance - it's a really obscure opam corner!